crates/sound/src/audio_backends/pw_backend.rs

```diff
@@ -4,23 +4,12 @@
 use super::AudioBackend;
 use std::{thread};
 use std::{cell::Cell, rc::Rc};
-use crate::{Error, Result};
+use crate::{Result};
 
-use vm_memory::{Le32, Le64};
+use vm_memory::{Le32};
 use pipewire as pw;
 use pw::{sys::*};
```

Comment:
I would really recommend not importing everything from the sys crate, it makes it very difficult to track from where a function comes from. calling `pw::sys::some_ffi_func` is not that bad, or at worst, just explicitly import whatever you need